### PR TITLE
Payment Wizard Fieldset

### LINKED ISSUE #1967: Add field set support to the Payment Wizard
Hub request: https://powerofus.force.com/0D58000002WX0ii

User would like to add some of their custom fields to the payments section of the wizard.

### LINKED ISSUE #2588: Payment Wizard interval frequency picklist values should be labels, not hard coded
In the Payment Wizard, the interval drop down selection is hard-coded, which prevents non-English users from translating the values. 
salesforce developer edition 1

With API 39.0 we have the ability to reference picklist API names instead of labels in our code, so we may want to look into that as a solution that is integration proof and localization proof.

### LINKED ISSUE #2620: Payment wizard error if Date of First Payment cleared
bring up the Payment wizard.
clear out the date from the Date of First Payment field
click on the Calculate Payments button

Attempt to de-reference a null object
Error is in expression '{!calculate}' in component <apex:commandButton> in page pmt_paymentwizard: Class.djhcumulus2.PMT_PaymentWizard_CTRL.calculate: line 302, column 1

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes
- Thank you to Thomas Taylor for adding the ability to use a field set on the Payment Wizard page. You can now customize the fields that are displayed when generating payments by updating the Payment_WizardFS field set. Note that field sets are only available in Salesforce Classic interface.

# Issues Closed
Fixes #1967
Fixes #2588
Fixes #2620

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by jjbennett** [comment]
@smarovich - This PR will be for the next release, can you please review the Changes section when you have a chance.
@force2b can you pair up with @joy-sfdo on this review? Joy is QE but wanted to also get your eyes on it with all of the work you had recently done on this page. This would be for the next release.

**Comment by force2b** [comment]
@jjbennett Should this PR list the 3 issues that Tommy lists as fixed? https://github.com/SalesforceFoundation/Cumulus/pull/2692

**Comment by jjbennett** [comment]
@force2b yes! updated the PR, thank you!

**Comment by smarovich** [comment]
@jjbennett Can you confirm the name of the field set? Is it Payment_WizardFS?  I was going to mention it in the release notes.

**Comment by jjbennett** [comment]
@smarovich yes, Payment_WizardFS is correct, thanks!

**Comment by smarovich** [comment]
Great. Thanks for confirming! I updated the release notes.

File: src/pages/PMT_PaymentWizard.page [hunk 1]
```diff
@@ -27,295 +23,287 @@ Copyright (c) 2008 Evan Callahan, evanc@npowerseattle.org, NPower Seattle, 403 2
...
+                                            <apex:outputText value="{!p.paymentNumber}"/>
+                                        </div>
+                                    </td>
+                                    <apex:repeat value="{!pmtFieldSet}" var="f"> 
+                                        <td>
+                                            <c:UTIL_FormField field="{!f.fieldPath}"
+                                                              sObj="{!p.OppPayment}"
+                                                              sObjType="npe01__OppPayment__c"
+                                                              showLabel="false"
+                                                              required="!f.DBRequired || f.Required}" />
```
Comment (force2b) [inline src/pages/PMT_PaymentWizard.page]: The value in the required attribute looks like it might be missing an opening curly bracket.

Comment (jjbennett) [reply src/pages/PMT_PaymentWizard.page]: @force2b good catch, fixed, thank you!

File: src/translations/fr.translation [hunk 2]
```diff
@@ -1824,6 +1828,10 @@ Pour vérifier vos paramètres d’allocations UCG, accédez à Dons | Allocatio
         <name>pmtWizardWarning</name>
     </customLabels>
     <customLabels>
+        <label>Semaine</label>
+        <name>pmtWizardWeek</name>
+    </customLabels>
+    <customLabels>
         <label>Quelle date voulez-vous utiliser ?</label>
```
Comment (joy-sfdo) [inline src/translations/fr.translation]: Remove extra space before "?".

**Comment by jjbennett** [comment]
@joy-sfdo nice catch, fixed!

**Comment by joy-sfdo** [comment]
@jjbennett -- Excellent! Thanks!

**Pull request merged**